h2o-core/src/main/java/hex/Model.java

```diff
@@ -1204,7 +1206,6 @@ else if (isOffset)
         else if (isWeights && computeMetrics) {
           if (expensive) {
             vec = test.anyVec().makeCon(1);
-            toDelete.put(vec._key, "adapted missing vectors");
```

Comment:
this looks suspicious, why is it okay to not delete the key?

Reply:
I accidentally deleted it. Fortunately, we have reviews. :)